manifests/h/HiroSystems/Clarinet/0.14.1/HiroSystems.Clarinet.installer.yaml

```diff
@@ -0,0 +1,13 @@
+PackageIdentifier: HiroSystems.Clarinet
+PackageVersion: 0.14.1
+Installers: 
+- InstallerLocale: en-US
+  Architecture: x64
+  InstallerType: msi
+  InstallerUrl: https://github.com/hirosystems/clarinet/releases/download/v0.14.1/clarinet-0.14.1-x86_64.msi
+  InstallerSha256: 311162885122C8EE018DAF46EBBE283C6626DBBA4A426627F8AE1732385DBC87
+  InstallerSwitches:
+    Silent: /quiet
+    SilentWithProgress: /passive
```

Comment:
```suggestion
```

Reply:
These aren't needed here; the client determines which switches to use based on `InstallerType`.